src/util/ConfigManager/ConfigManager.cpp

```diff
@@ -690,12 +765,71 @@ void ConfigManager::verifyWithValidators() const {
 
 // ____________________________________________________________________________
 bool ConfigManager::containsOption(const ConfigOption& opt) const {
-  const auto allOptions = configurationOptions(false);
+  const auto allOptions = configurationOptions();
   return ad_utility::contains(
       std::views::values(allOptions) |
           std::views::transform(
               [](const ConfigOption& option) { return &option; }),
       &opt);
 }
 
+// ____________________________________________________________________________
+template <typename T>
+requires std::same_as<T, ConfigOption> || std::same_as<T, ConfigManager>
```

Comment:
I think we have `ad_utility::TypeContainedIn`.

Reply:
Done, see [5859624](https://github.com/ad-freiburg/qlever/pull/1169/commits/5859624b3c6c9b59d50cf1ade82d73977d0d6f55).